### ISSUE TITLE
Trailing comma doesn't affect the formatting of assert()

### ISSUE THREAD

**Issue by DanTup** [dart-lang/dart_style#745]
(Originally raised at https://github.com/Dart-Code/Dart-Code/issues/1273 by @apaatsio)

To reproduce:

1. Write line: `assert(foo != null, 'hello',);` (Note the trailing comma after 'hello')
2. Run "Format Document"

Expected output:

```dart
assert(
  foo != null,
  'hello',
);
```

Actual output:
```dart
assert(foo != null, 'hello',);
```

**Comment by munificent** [comment]
Ugh, yet another trailing comma corner case.

**Comment by a14n** [comment]
Fixed via #790

**Comment by munificent** [comment]
That's right, thanks @a14n!
